widgetssdk/src/main/res/layout/survey_view.xml

```diff
@@ -4,72 +4,77 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:parentTag="androidx.constraintlayout.widget.ConstraintLayout">
 
-    <androidx.cardview.widget.CardView
-        android:id="@+id/card_view"
-        android:layout_height="match_parent"
+    <androidx.cardview.widget.CardView xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/cut_bottom_corners_card_view"
         android:layout_width="match_parent"
-        android:backgroundTint="@color/glia_base_light_color"
-        android:layout_marginTop="@dimen/glia_x_large" >
-
-        <androidx.constraintlayout.widget.ConstraintLayout
-            android:id="@+id/content_view"
+        android:layout_height="match_parent"
+        card_view:cardBackgroundColor="@android:color/transparent"
+        card_view:cardCornerRadius="0dp">
+        <!-- Solution for making CardView only have corner radius at the top -->
+        <androidx.cardview.widget.CardView
+            android:id="@+id/card_view"
+            android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:layout_width="match_parent">
+            card_view:cardCornerRadius="24dp">
```

Comment:
`glia_survey_default_survey_corner_radius` ?

Reply:
Fixed in 45cb829694f68426f3ae9bb022434a6b9c255792
Thank you